src/PluginStructs/P200_data_struct.cpp

```diff
@@ -98,38 +98,49 @@ bool P200_data_struct::loop() {
           ++sentences_received_error;
         } else {
           // JSON & CSV
-          String field = "";
+          String field;
           uint16_t field_start = 2;
           uint16_t field_end = sentence_part.indexOf('\r', field_start);
           uint16_t field_separator = 0;
-          if (output_type == P200_OUTPUT_JSON) { last_sentence = '{'; }
+          if (output_type == P200_OUTPUT_JSON) { last_sentence = F("{"); }
```

Comment:
Actually, a char is 'lighter'/'cheaper' than a singl-character flash-string, so the previous code was just fine.
OTOH, there are also global helper functions to wrap a string value with quotes, and to output a single json name/value item.

Reply:
Changed back to char.